tests/integration/index.test.ts

```diff
@@ -82,7 +75,9 @@ describe('Process creation and deletion', () => {
     })
 
     it('fails for invalid json', async () => {
-      return assert.isRejected(createProcess(validProcessName, validVersionNumber, 'invalidJson'), SyntaxError)
+      // ts is robust for this. I must use any in order for this test to work
+      //Argument of type 'string' is not assignable to parameter of type 'Program'.ts(2345)
+      return assert.isRejected(createProcess(validProcessName, validVersionNumber, 'invalidJson' as any), TypeError)
```

Comment:
I don't mind the error, but maybe you can include an eslint ignore line to ignore this specific linting warning

Reply:
```
// eslint-disable-line @typescript-eslint/no-explicit-any
```
?

Reply:
So, the function basically won't work unless I pass it as a member of Program, because tests are strongly types as well, this is one of the reasons why I like to have my tests written in js without strong types, gives me also way more flexibility and I can focus on scenarios and coverage rather than typing my tests. Compiler checks as well, and it's nice to have tests that break stuff...